apps/system/js/cards_helper.js

```diff
@@ -32,8 +23,16 @@
 
       sizes.sort(function(x, y) { return y - x; });
 
-      var index = sizes[(HVGA) ? sizes.length - 1 : 0];
-      iconPath = icons[index];
+      iconPath = icons[0]; // The biggest icon available
+      for (var i = 0; i < sizes.length; i++) {
+        var size = sizes[i];
+
+        if (size < maxSize) {
```

Comment:
Well, you would still use it. If you have a <= then you would end up breaking, and not assigning the 60 image to the iconPath after the if statement. You would exit on the next iteration, and preserve the 60.

The logic is a bit confusing, if you have any ideas about how to clean it up, let me know.